mobile/Ihc/models/Patient.js

```diff
@@ -100,11 +100,11 @@ export default class Patient {
   static getInstance(lastUpdated = new Date().getTime(),
     key = 'firstname&father&mother&20000101', firstName = 'firstname',
     fatherName = 'father', motherName = 'mother', birthday = '20000101',
-    gender = 1, phone = null, motherHeight = 100, fatherHeight = 100,
-    drugUpdates = [], soaps = [], triages = [], statuses = []) {
+    gender = 1, phone = null, drugUpdates = [], soaps = [], triages = [],
```

Comment:
Why are you removing motherHeight and fatherHeight?

Reply:
I asked IHC about it and they weren't sure why it was there.  They said it might be useful if we're distinguishing between kids and adults, but it doesn't seem useful for now

Reply:
Ok sg. This is one of those things that should be submitted separately because its not related to any of the other stuff

Reply:
So, how should I submit separately? Make a new branch?

Reply:
its okay for this pull request cause git is too complicated... just keep in mind for the future